results/scripts/batch_convert_results.py

```
@@ -0,0 +1,131 @@
+        sys.executable,  # Use the same Python interpreter
+        script_path,
+        benchmarks_json,
+        "--metadata-file", metadata_json,
+        "--csv-file", output_csv,
+    ]
+
+    print(f"\nProcessing: {benchmarks_json}")
+    print(f"  Metadata: {metadata_json}")
+
+    try:
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)